pipelines/delta-cleanup.js

```diff
@@ -0,0 +1,61 @@
+import { querySudo as query, updateSudo as update } from '@lblod/mu-auth-sudo';
+import {createError} from "../lib/error";
+import {DELTA_FILES_RETENTION_PERIOD, JOBS_GRAPH, SERVICE_NAME} from "../config";
+import {PREFIXES} from "../lib/constants";
+import DeltaFile from "../lib/delta-file";
+export async function startDeltaCleanup() {
+    try {
+        // a retention period of -1 indicates files should not be removed
+        if (DELTA_FILES_RETENTION_PERIOD !== -1) {
```

Comment:
I would change it to `if (DELTA_FILES_RETENTION_PERIOD > 0) {` otherwise what if we set -2? if -2 should work, then -1 should also work.